specification/monitor/resource-manager/Microsoft.Monitor/preview/2021-06-03-preview/operations_API.json

```diff
@@ -52,7 +52,7 @@
           "Operations"
         ],
         "description": "Lists available Operations for this Resource Provider",
-        "operationId": "Monitor_Operations_List",
+        "operationId": "Monitor_OperationsList",
```

Comment:
Linter errors need to be fixed related to this. Since operation response has model definition in array type, it should be of the form '_list'.

Reply:
Sorry about that. I have updated the value to "MonitorOperations_List" in the most recent commit, e44fed6